src/tests/ftest/pool/rebuild_with_io.py

```diff
@@ -20,143 +20,93 @@
   provided in Contract No. B609815.
   Any reproduction of computer software, computer software documentation, or
   portions thereof marked with this legend must also reproduce the markings.
-'''
-from __future__ import print_function
+"""
+from apricot import TestWithServers, skipForTicket
+from test_utils import TestPool, TestContainer
 
-import os
-import time
-import traceback
 
-from apricot import TestWithoutServers
+class RebuildWithIO(TestWithServers):
+    """Test class for pool rebuild during I/O.
 
-import agent_utils
-import server_utils
-import write_host_file
-import check_for_pool
-import io_utilities
-from daos_api import DaosPool, DaosServer, DaosContainer, DaosApiError
+    Test Class Description:
+        This class contains tests for pool rebuild that feature I/O going on
+        during the rebuild.
 
-class RebuildWithIO(TestWithoutServers):
-    """
-    This class contains tests for pool rebuild that feature I/O going on
-    during the rebuild.
     :avocado: recursive
     """
-    def __init__(self, *args, **kwargs):
-        super(RebuildWithIO, self).__init__(*args, **kwargs)
-        self.agent_sessions = None
 
+    @skipForTicket("DAOS-2922")
     def test_rebuild_with_io(self):
-        """
-        Test ID: Rebuild-003
+        """JIRA ID: Rebuild-003.
 
-        Test Description: Trigger a rebuild while I/O is ongoing.
+        Test Description:
+            Trigger a rebuild while I/O is ongoing.
 
         Use Cases:
-          -- single pool, single client performing continous read/write/verify
-             sequence while failure/rebuild is triggered in another process
+            single pool, single client performing continous read/write/verify
+            sequence while failure/rebuild is triggered in another process
 
         :avocado: tags=pool,rebuild,rebuildwithio
```

Comment:
May be you forgot to include those tags in this test

Reply:
Thanks.  The latest patch includes the tags

Reply:
I should have been more clear in my comments. As you have included "all,pr,medium" in other tests could you please also include it for this one too. Once you submit the patch this test should run as well in CI then and we will be more sure that it works fine in CI.

Reply:
I've updated the rebuild_with_io.py avocado tags as well as corrected their inclusion in the test tags list in the commit message.